files/entrypoint.sh

```diff
@@ -34,6 +34,9 @@ if [ -f /firstrun ]; then
 	# Configure WEB UI access
 	sed -ie "s/^\$Conf{CgiAdminUsers}\s*=\s*'\w*'/\$Conf{CgiAdminUsers} = '${BACKUPPC_WEB_USER:-backuppc}'/g" /etc/backuppc/config.pl
 	htpasswd -b -c /etc/backuppc/htpasswd ${BACKUPPC_WEB_USER:-backuppc} ${BACKUPPC_WEB_PASSWD:-password}
+	
+	# Workaround bug which causes BackupPC fail to start when container launched with existing configuration
```

Comment:
For future readers, you should say in the comment that this issue happens during a migration from BackupPC v3 to BackupPC v4.

Reply:
Please note that the issue is not about updating from v3 -> v4.

It is about starting the same v3 container again - and it overwrites configuration (see description of this PR).

Reply:
Yes, so the comment, written exclusively on the v3, gives enough context.